packages/key-management/src/util/ownSignatureKeyPaths.ts

```diff
@@ -3,6 +3,52 @@ import { Cardano } from '@cardano-sdk/core';
 import { isNotNil } from '@cardano-sdk/util';
 import uniq from 'lodash/uniq';
 
+/**
+ * Gets whether any certificate in the provided certificate list requires a stake key signature.
+ *
+ * @param rewardAccounts The known reward accounts.
+ * @param certificates The list of certificates.
+ */
+// eslint-disable-next-line complexity
+const isStakingKeySignatureRequired = (
+  rewardAccounts: Cardano.RewardAccount[],
```

Comment:
```suggestion
  rewardAccounts: Cardano.RewardAccount,
```
This function should either:
- check only 1 reward account - this is perfectly fine for now since we're always signing with the same 1 stake key
- return stake key paths to sign with

Reply:
No reason, just simpler. Supporting multiple stake keys in this function would be nice, but then we need to map those keys to derivation paths to know what to sign with.